docs/how-to-guides/tracing/index.md

```diff
@@ -34,18 +36,7 @@ completion = client.chat.completions.create(
 print(completion.choices[0].message)
 ```
 
-Running above python script will produce below example output:
-```
-Prompt flow service has started...
-You can view the traces from local: http://localhost:<port>/v1.0/ui/traces/?#collection=basic
-```
-
-Click the trace url, user will see a trace list that corresponding to each LLM calls:
-![LLM-trace-list](../../media/trace/LLM-trace-list.png)
-
-
-Click on one line record, the LLM detail will be displayed with chat window experience, together with other LLM call params:
-![LLM-trace-detail](../../media/trace/LLM-trace-detail.png)
+Then OpenAI is instrumented, and as prompt flow follows OpenTelemetry specification, user can fully leverage the OpenTelemetry knowledge to use these traces during the OpenAI calls.
```

Comment:
Why removed the result pictures for the very first example?

Reply:
screenshots are related to trace UI, and we have a separate page for it, so moved these to that page.

Reply:
Yeah but you give UI piciture soon in the next section `Trace LLM and frameworks` without much introduction, may cause some confusion.

Reply:
let me add some explaination on those screenshots